source/dataman/AMScanDictionary.cpp

```diff
@@ -218,11 +218,8 @@ QString AMScanDictionary::krFacilityName(const QString& arg) {
 	if(!scan_)
 		return "[??]";
 
-	AMRun r;
-	r.loadFromDb(scan_->database(), scan_->runId());
-
 	AMFacility f;
-	f.loadFromDb(scan_->database(), r.facilityId());
+	f.loadFromDb(scan_->database(), 1); //AMFacility table will have only one entry, which is the definition of this facility
```

Comment:
@dretrex I think from this point forward, we will have only one facility in each DB and this is also what we are trying to do here (if this assumption doesn't stand, then we need to rework on the DBUpgrade and this piece of code).

Regarding BioXAS, I talked with @helfrij and just pushed one changes 29f48754de090a351824e540c2f339c2fe5449b4, which will make the BioXAS only one facility.